<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QDialog" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Everpad / All Notes</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="newNotebookBtn">
           <property name="toolTip">
            <string>Create Notebook</string>
           </property>
           <property name="text">
            <string>Notebook</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newNoteBtn">
           <property name="toolTip">
            <string>Create Note</string>
           </property>
           <property name="text">
            <string>Note</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="EverpadTreeView" name="notebooksList">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="notesList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EverpadTreeView</class>
   <extends>QTreeView</extends>
   <header>everpad/pad/treeview</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
